build/pipelines/templates/build-console-ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   variables:
     BuildConfiguration: ${{ parameters.configuration }}
     BuildPlatform: ${{ parameters.platform }}
-  pool: { vmImage: vs2017-win2016 }
+  pool: { vmImage: windows-2019 }

```

Comment:
this change should be also done to `build-console-audit-job.yml` and `check-formatting.yml`